### PR TITLE
Add footer section

### PR DESCRIPTION
**Pull request by jfranciscosousa**
Also refactors separator width variable

### CODE REVIEW

File: src/components/Footer/index.css [hunk 1]
```diff
@@ -0,0 +1,206 @@
+@import "../../styles/all.css";
+
+$social-icon-size: 28px;
+$social-icon-size-mobile: 16px;
```
Comment (filipebfernandes) [inline src/components/Footer/index.css]: Even if those are icon sizes, shouldn't we use spacings?

File: src/components/Footer/index.css [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+  display: flex;
+
+  @include Breakpoint-tabletAndAbove {
+    justify-content: space-between;
+    width: 100%;
+    max-width: $three-columns;
+  }
+
+  @include Breakpoint-mobileOnly {
+    flex-direction: row;
```
Comment (filipebfernandes) [inline src/components/Footer/index.css]: I think you don't need this mobile breakpoint

File: src/components/Footer/index.css (hunk 1 above)
Comment (jfranciscosousa) [reply src/components/Footer/index.css]: Spacings are space between thing, image sizes are other (sizes can be related in some cases) aspect

Comment (jfranciscosousa) [reply src/components/Footer/index.css]: But i can use spacings in fact, in this case there is a correlation, but on the team photos for example, not so much

Comment (filipebfernandes) [reply src/components/Footer/index.css]: 👍

**Pull request merged**